CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 - Added: `consecutive-duplicates` option for `declaration-block-no-duplicate-properties` rule.
 - Fixed: `block-no-empty` no longer delivers false positives for less syntax.
+- Fixed: `CssSyntaxError` is no longer thrown but reported alongside warnings.
```

Comment:
Can you move this to the top of the CHANGELOG and change the type to "Changed" please? i.e. 

- Changed: `CssSyntaxError` is no longer thrown but reported alongside warnings.

"Changed" and "Removed" indicate a breaking change for us.